src/bin/cargo/commands/owner.rs

```diff
@@ -33,19 +83,56 @@ pub fn cli() -> Command {
         ))
 }
 
+fn add_registry_args(command: Command) -> Command {
+    command
+        .arg_index("Registry index URL to modify owners for")
+        .arg_registry("Registry to modify owners for")
+        .arg(opt("token", "API token to use when authenticating").value_name("TOKEN"))
+}
+
 pub fn exec(config: &mut Config, args: &ArgMatches) -> CliResult {
+    let (to_add, to_remove, list) = match args.subcommand() {
+        Some(("add", subargs)) => (
+            subargs
+                .get_many::<String>("add")
+                .map(|xs| xs.cloned().collect::<Vec<String>>()),
+            None,
+            false,
+        ),
+        Some(("remove", subargs)) => (
+            None,
+            subargs
+                .get_many::<String>("remove")
+                .map(|xs| xs.cloned().collect()),
+            false,
+        ),
+        Some(("list", _)) => (None, None, true),
+        Some((name, _)) => {
+            unreachable!("{name} is not a subcommand of cargo owner, please enter `cargo owner --help` for help.")
+        }
```

Comment:
Can we hit this?

If clap is preventing it, thats what the panic should be about

If clap is not, then we should have a user-facing error message, not a panic

Reply:
Okay, now I'm getting an error back.